src-plugins/vtkDataMesh/writers/vtkDataMeshWriter.cpp

```diff
@@ -53,8 +59,6 @@ bool vtkDataMeshWriter::write(const QString& path)
   if (!this->data())
     return false;
 
-  qDebug() << "Can write with: " << this->identifier();
-
   medAbstractData * medData = dynamic_cast<medAbstractData*>(this->data());
```

Comment:
In `vtkDataMeshWriter::getHeaderVtk()` and `vtkDataMeshWriter::addHeaderVtpToMesh` i access to `this->data()->metadata(key)`. But vtkMetaSurfaceMesh is created through `vtkMetaDataSet * mesh = dynamic_cast< vtkMetaDataSet*>( (vtkObject*)(this->data()->data()));` which lose every metadata.